cpp/include/raft/neighbors/specializations.cuh

```diff
@@ -16,6 +16,12 @@
 
 #pragma once
 
+<<<<<<< HEAD
```

Comment:
Looks like this needs to be merged

Reply:
We want to move the `ball_cover` and `fused_l2_knn` specializations to `raft/spatial/knn/specializations.cuh` btw, since they are only present in `libraft-nn` at the moment